client/apps/websocket/index.js

```diff
@@ -0,0 +1,92 @@
+import { messageTypes } from './messageTypes'
+
+const {
+  articlesRequested,
+  articleLocked,
+  userStartedEditing,
+  userStoppedEditing,
+  userCurrentlyEditing
+} = messageTypes
+
+export const articlesInSession = {}
+
+export const onArticlesRequested = ({io, socket}) => {
+  console.log('[socket] onArticlesRequested')
+  const event = articlesRequested
+  socket.emit(event, {
+    type: 'EDITED_ARTICLES_RECEIVED',
+    payload: articlesInSession
+  })
+}
+
+export const onUserStartedEditing = ({io, socket}, data) => {
+  console.log('[socket] onUserStartedEditing, userId: ', data.user.id, ', article: ', data.article)
+  const { timestamp, user, article } = data
```

Comment:
Stray console log, or do you want this here?

Reply:
the `[socket]` logs are intented to be there for server debugging purposes